site/components/form/DateSelector.tsx

```diff
@@ -147,19 +133,7 @@ const DateSelector = <T extends object>({
                         }}
                         onAccept={() => setErrors([])}
                         renderInput={({ inputRef, inputProps, InputProps }) => {
-                            return inputBox(
-                                inputRef,
-                                inputProps,
-                                InputProps,
-                                inputId,
-                                inputName,
-                                errors,
-                                disabled,
-                                stateUpdater,
-                                setErrors,
-                                schema,
-                                errorOnBlur,
-                            );
+                            return inputBox(inputRef, inputProps, InputProps, inputId, inputName, errors, disabled);
```

Comment:
RangeError is thrown when you submit a new disruption without date and time fields.

Reply:
@alvak1 This is actually a bug on main, I've fixed it in my PR

Reply:
pulled main in the fix is now in